avocado-setup.py

```diff
@@ -266,6 +267,18 @@ def install_optional_plugin(plugin):
         # plugin already installed
         pass
 
+def configure_prepostscript(repos, scripttype):
+    i=0
+    cmd= 'mkdir -p /etc/avocado/scripts/job/%s.d' % scripttype
+    helper.runcmd(cmd, ignore_status=True, info_str="%s script directory created" % scripttype)
+    for repo in repos:
+        i=i+1
```

Comment:
We can do something using zip() or map() instead of manually computing iteration values

Reply:
ah that makes sense